docs/changes.rst

```diff
@@ -155,13 +155,19 @@ Compatibility notes:
 - Repositories in the "authenticated" mode are now treated as the unencrypted
   repositories they are.
 
+- The client-side repository cache now holds unencrypted data for better speed.
```

Comment:
Perhaps "temporary repository cache" to make it more clear that this isn't about _the_ cache.